openid-connect-server/src/main/java/org/mitre/openid/connect/service/impl/MITREidDataService_1_1.java

```diff
@@ -318,13 +318,13 @@ private void writeGrants(JsonWriter writer) throws IOException {
             writer.name("allowedScopes");
             writeNullSafeArray(writer, site.getAllowedScopes());
             writer.name("whitelistedSiteId").value(site.getIsWhitelisted() ? site.getWhitelistedSite().getId() : null);
-            Set<OAuth2AccessTokenEntity> tokens = site.getApprovedAccessTokens();
-            writer.name("approvedAccessTokens");
-            writer.beginArray();
-            for (OAuth2AccessTokenEntity token : tokens) {
-                writer.value(token.getId());
-            }
-            writer.endArray();
+//            Set<OAuth2AccessTokenEntity> tokens = site.getApprovedAccessTokens();
+//            writer.name("approvedAccessTokens");
+//            writer.beginArray();
+//            for (OAuth2AccessTokenEntity token : tokens) {
+//                writer.value(token.getId());
+//            }
+//            writer.endArray();
             writer.endObject();
```

Comment:
Again, just like the above, this is dropping data on the floor without replacing it. You still need to export the approved->token links, and you can't actually change the export format like this. (The export can have fields added but you can't remove fields like this without breaking version compatibility.)